### PR TITLE
DB-1362: Merge with released version FF 57

### PR DESCRIPTION
**Pull request by alver-cliqz**

### CODE REVIEW

File: mozilla-release/CLOBBER [hunk 1]
```diff
@@ -22,4 +22,4 @@
 # changes to stick? As of bug 928195, this shouldn't be necessary! Please
 # don't change CLOBBER for WebIDL changes any more.
 
-Bug 1399970 / Bug 1399226 - Removed some now-unused icons. Clobbering to avoid `make package` failures, e.g., bug 1399686.
+Merge day clobber
```
Comment (chrmod) [inline mozilla-release/CLOBBER]: can we remove this file now?

File: mozilla-release/accessible/windows/msaa/Compatibility.cpp [hunk 2]
```diff
@@ -168,7 +168,7 @@ Compatibility::Init()
 
   HMODULE jawsHandle = ::GetModuleHandleW(L"jhook");
   if (jawsHandle)
-    sConsumers |= (IsModuleVersionLessThan(jawsHandle, 18, 4315)) ?
+    sConsumers |= (IsModuleVersionLessThan(jawsHandle, 19, 0)) ?
```
Comment (chrmod) [inline mozilla-release/accessible/windows/msaa/Compatibility.cpp]: what is that?

File: mozilla-release/browser/components/nsBrowserGlue.js [hunk 3]
```diff
@@ -1731,7 +1731,7 @@ BrowserGlue.prototype = {
 
   // eslint-disable-next-line complexity
   _migrateUI: function BG__migrateUI() {
-    const UI_VERSION = 57;
+    const UI_VERSION = 58;
```
Comment (chrmod) [inline mozilla-release/browser/components/nsBrowserGlue.js]: looks like this should be 57, right?

File: mozilla-release/browser/extensions/formautofill/FormAutofillContent.jsm [hunk 4]
```diff
@@ -108,6 +108,7 @@ AutofillProfileAutoCompleteSearch.prototype = {
                           FormAutofillUtils.isAutofillAddressesEnabled :
                           FormAutofillUtils.isAutofillCreditCardsEnabled;
 
+    ProfileAutocomplete.lastProfileAutoCompleteFocusedInput = focusedInput;
```
Comment (chrmod) [inline mozilla-release/browser/extensions/formautofill/FormAutofillContent.jsm]: guess all this does not matter as we don't enable autofill extension at all

**Comment by alver-cliqz** [comment]
@chrmod all this changes from released FF 57 version. https://hg.mozilla.org/releases/mozilla-release/rev/FIREFOX_57_0_RELEASE. Previously we works with the FIREFOX_57_BASE tag. So, this is what original released version have.

**Pull request merged**